data/wanderers start.txt

```diff
@@ -501,6 +501,9 @@ ship "Deep River" "Deep River (Jump, Unarmed)"
 		"Type 4 Radiant Steering"
 		"Jump Drive"
 
+phrase "deep river"
```

Comment:
Would it be better to name this is a way that references the mission? Or just use the generic name generator?

Reply:
The generic name generator can't be used because this ship is in two missions, and the ship needs to be altered between the missions (it loses its cargo). I thought about making the phrase name more specific, but it doesn't really matter right now unless we need to do this again.

Reply:
In fairness, the return trip (empty) needs no special treatment. So really you are only jumping hoops on the should-have-cargo mission legs